test/e2e/tests/hardware-wallets/trezor-account.spec.js

```diff
@@ -92,4 +92,98 @@ describe('Trezor Hardware', function () {
       },
     );
   });
+
+  it('unlocks multiple accounts at once', async function () {
+    await withFixtures(
+      {
+        fixtures: new FixtureBuilder().build(),
+        ganacheOptions: defaultGanacheOptions,
+        title: this.test.fullTitle(),
+      },
+      async ({ driver }) => {
+        await unlockWallet(driver);
+        await connectTrezor(driver);
+
+        // Unlock 5 Trezor accounts
+        const accountCheckboxes = await driver.findElements(
+          '.hw-account-list__item__checkbox',
+        );
+        await accountCheckboxes[0].click();
+        await accountCheckboxes[1].click();
+        await accountCheckboxes[2].click();
+        await accountCheckboxes[3].click();
+        await accountCheckboxes[4].click();
+
+        await driver.clickElement({ text: 'Unlock' });
+
+        // Check that all 5 Trezor accounts are present
+        await driver.clickElement('[data-testid="account-menu-icon"]');
+        for (let i = 0; i < 5; i++) {
+          assert(
+            await driver.isElementPresent({
+              text: `Trezor ${i + 1}`,
+            }),
+            `Trezor account ${i + 1} not found`,
+          );
+          assert(
+            await driver.isElementPresent({
+              text: shortenAddress(KNOWN_PUBLIC_KEY_ADDRESSES[0].address),
```

Comment:
should it be `KNOWN_PUBLIC_KEY_ADDRESSES[i].address` ?

Reply:
indeed! Nice catch thank you :heart_eyes: